src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/model/PlacementApplicationReportRow.kt

```diff
@@ -44,4 +44,8 @@ data class PlacementApplicationReportRow(
   val notArrivedReason: String?,
   val placementRequestType: String?,
   val paroleDecisionDate: LocalDate?,
+  val assessmentAppealCount: Int?,
+  val lastAssessmentAppealedDecision: String?,
+  val lastAssessmentAppealedDate: LocalDate?,
+  val assessmentAppealedFromStatus: String?,
```

Comment:
perhaps I'm misunderstanding this report, but my understanding is that only assessments with the `rejected` decision are ever appealed? i.e. that this column must be definition always contain `rejected`?

Reply:
It always contains the original assessment decision, so when there's no appeal, it's the same as the `assessmentDecision` column. When an appeal gets made, `assessmentDecision` contains the decision of the most recent appeal for that assessment, and `assessmentAppealedFromStatus` will always be `"rejected"` in that case (for the reason you mentioned).